istioctl/cmd/proxyconfig.go

```diff
@@ -43,6 +48,101 @@ var (
 	clusterName, status string
 )
 
+// Level is an enumeration of all supported log levels.
+type Level int
+
+const (
+	defaultLoggerName  = "level"
+	defaultOutputLevel = WarningLevel
+)
+
+const (
+	// OffLevel disables logging
+	OffLevel Level = iota
+	// CriticalLevel enables critical level logging
+	CriticalLevel
+	// ErrorLevel enables error level logging
+	ErrorLevel
+	// WarningLevel enables warning level logging
+	WarningLevel
+	// InfoLevel enables info level logging
+	InfoLevel
+	// DebugLevel enables debug level logging
+	DebugLevel
+	// TraceLevel enables trace level logging
+	TraceLevel
+)
+
+// existing sorted active loggers
+var activeLoggers = []string{
```

Comment:
Is it possible to get these loggers from Envoy code rather than having our own version of the list?

Reply:
It's a cpp language macro.

https://github.com/envoyproxy/envoy-wasm/blob/a62329130853328a346f57ee23ed66aabefbfe22/source/common/common/logger.h#L24-L65